src/main/java/com/robohorse/robopojogenerator/generator/PostProcessorFactory.java

```diff
@@ -16,6 +19,13 @@ public PostProcessorFactory() {
     }
 
     public AbsPostProcessor createPostProcessor(GenerationModel generationModel) {
+
+        // Not support AutoValue yet
+        if (generationModel.getLanguageItem().equals(LanguageItem.KOTLIN_DTO)
+                && !generationModel.getAnnotationItem().equals(AnnotationItem.AUTO_VALUE_GSON)){
+            return Injector.getAppComponent().newKotlinDataClassPostProcessor();
+        }
+
```

Comment:
Emm, but what if I check the Kotlin checkbox and click AutoValue radio ?

Actually, Kotlin data class do almost the same things compare with AutoValue.
> Generate the properties/fields, `toString()`, `equals()`, `hashCode()`

I prefer to disable the AutoValue when choose Kotlin.

What's your opinion？